### ISSUE TITLE
heketi cannot remove devices which crashed

### ISSUE THREAD

**Issue by dmesser** [heketi/heketi#733]
When dealing with a situation where a device of a heketi cluster fails because of a disk error heketi cannot remove/delete that device in order to recover on another available brick.

An attempt to delete it will yield the following:
```

[ec2-user@ip-10-20-4-180 ~]$ heketi-cli device remove 5e0fc28d078938f7a1c54985846db962
Error: Failed to remove device, error: Unable to kill brick process 10.20.6.33:/var/lib/heketi/mounts/vg_5e0fc28d078938f7a1c54985846db962/brick_c359bcef310571ec9c16c6b570808e0b/brick
```

When a disk on a Gluster node fails the resulting errors from the XFS filesystem are propagated to the parent translator (see [feature description here](https://github.com/gluster/glusterfs-specs/blob/master/done/Features/brick-failure-detection.md)). 
As a result the brick process is killed in order to avoid hung state:

```
Apr 04 16:29:50 ip-10-20-6-33.us-west-2.compute.internal kernel: XFS (dm-4): xfs_log_force: error -5 returned.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Created slice user-0.slice.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Starting user-0.slice.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Started Session 242 of user root.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Starting Session 242 of user root.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Started Session 243 of user root.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Starting Session 243 of user root.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal CROND[1570]: (root) CMD (/usr/lib64/sa/sa1 1 1)
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal CROND[1569]: (root) CMD (/usr/lib64/sa/sa1 1 1)
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Removed slice user-0.slice.
Apr 04 16:30:01 ip-10-20-6-33.us-west-2.compute.internal systemd[1]: Stopping user-0.slice.

Broadcast message from systemd-journald@ip-10-20-6-33.us-west-2.compute.internal (Tue 2017-04-04 16:30:20 EDT):

var-lib-heketi-mounts-vg_5e0fc28d078938f7a1c54985846db962-brick_c359bcef310571ec9c16c6b570808e0b-brick[1420]: [2017-04-04 20:30:20.527708] M [MSGID: 113075] [posix-helpers.c:1847:posix_health_check_thread_proc] 0-vol_8f1ae194da1ab75b1c8660547adbe4c6-posix: still alive! -> SIGTERM
```

Therefore heketi cannot expect to be able to reach the brick process in all cases. There might be situations, such as the loss of a hard disk, where the brick process is likely to be gone. For these situations heketi should offer a --force switch to the device remove or delete operation. Without it impacted volumes cannot be recovered by the means of heketi and as such are subject to data loss in the long-term.

**Comment by MohamedAshiqrh** [comment]
https://github.com/heketi/heketi/pull/735

This will fix the issue.

**Comment by MohamedAshiqrh** [comment]
This is fixed we can close this. @lpabon @obnoxxx

**Comment by obnoxxx** [comment]
fixed by PR #735
